playbook-website/app/javascript/site_styles/docs/_markdown.scss

```diff
@@ -59,6 +59,11 @@
       transition: 0.1s all ease-in-out;
     }
   }
+  h4:last-of-type {
+    border-top: 1px solid $border_light;
+    padding-top: $space_md;
+    font-weight: $regular;
+  }
```

Comment:
Everything looks great @nickamantia , but I think you'll need to nest this inside the changelog classname as well. It's effecting the markdown styling for h4 that we have for the website
Here's what the guides/design_guidelines/example_markdown_styling page looks like on prod:
![Screenshot 2023-09-14 at 8 48 22 AM](https://github.com/powerhome/playbook/assets/73710701/80e4b47d-b233-450b-a41c-4d98e449c384)

And this is on your review env:
![Screenshot 2023-09-14 at 8 48 29 AM](https://github.com/powerhome/playbook/assets/73710701/20c9a205-5962-4982-8182-d1745a906abd)

cc: @jasperfurniss

Reply:
great catch! gonna fix this right now! thanks!!